go/libraries/doltcore/env/actions/docs.go

```diff
@@ -123,17 +125,17 @@ func getUpdatedWorkingAndStagedWithDocs(ctx context.Context, dEnv *env.DoltEnv,
 		root = staged
 	}
 
-	docs, err := dEnv.GetDocsWithNewerTextFromRoot(ctx, root, docDetails)
+	docs, err := env.GetDocsWithTextFromRoot(ctx, root, docDetails)
```

Comment:
This should be a method in the doltdocs package, takes a root and returns docs.

DocsInRoot